p2p/libp2p/metrics/connsDisconnsMetric.go

```diff
@@ -0,0 +1,54 @@
+package metrics
+
+import (
+	"sync/atomic"
+
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/multiformats/go-multiaddr"
+)
+
+// ConnsDisconnsMetric is a metric that counts connections and disconnections done by the host implementation
+type ConnsDisconnsMetric struct {
+	numConnections    uint32
```

Comment:
You are already in metrics package, Connection as struct name should be enough, as well as for file name

Reply:
Good point. Will rename to Connections

Reply:
done